doc/api/API.md

```diff
@@ -92,16 +97,19 @@ PdftronFlutter.setRequestedOrientation(0);
 ```
 
 ## Viewer Functions
+
```

Comment:
Why are there so many formatting changes to the API file? Seems like it should be a separate PR.
I think this is from before you took over, @terryyoon-pdftron. I guess it's ok.